lib/cabin/channel.rb

```diff
@@ -75,17 +75,34 @@ def each(&block)
       end
     end # def Cabin::Channel.each
 
+    # Get a list of actions included in this class.
+    def actions
+      @actions ||= []
+    end # def Cabin::Channel.actions
+
+    # Register a new filter. The block is passed the event. It is expected to
+    # modify that event or otherwise do nothing.
+    def action(&block)
+      @actions ||= []
+      @actions << block
+    end
+
     # Get a list of filters included in this class.
     def filters
       @filters ||= []
     end # def Cabin::Channel.filters
-    
-    # Register a new filter. The block is passed the event. It is expected to
-    # modify that event or otherwise do nothing.
+
+    # Register a new filter. The block is passed the event and the subscription.
+    # It is expected to either return true or false
```

Comment:
Maybe we should call this condition instead of filter, now that 'action' has been split out?